src/pages/UserTasks.tsx

```diff
@@ -59,6 +59,13 @@ export default function YourTasks() {
           setLoading(false)
           return
         }
+        const backendUserId = localStorage.getItem("backendUserId")
+        if (!backendUserId) {
+          setError("User ID not found in localStorage. Please sign up again.")
```

Comment:
where is this error set or shown? If to the user, User doesn't need to know about local storage.